pom.xml

```diff
@@ -30,6 +31,18 @@
       <artifactId>jetty-proxy</artifactId>
       <version>${jetty.version}</version>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+      <version>${junit.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.github.tomakehurst</groupId>
+      <artifactId>wiremock</artifactId>
+      <version>1.58</version>
```

Comment:
I see your point on level of indirection, but it is convenient to have **all** version numbers together. If we include them with each dependency then you're crawling through each dependency's XML block **a)** looking for the right block and **b)** looking for the version element.

Fortunately our number of dependencies is small so it isn't that big of a task. I'm open for either approach. @epugh has a point.